package.json

```diff
@@ -28,6 +28,6 @@
   },
   "dependencies": {
     "@rollup/pluginutils": "^3.1.0",
-    "esbuild": "^0.5.9"
+    "esbuild": "^0.6.3"
```

Comment:
What are your thoughts on moving this to a `peerDependency`? That way the version of esbuild is less-locked-in to this plugin.

Reply:
👌 that's probably better

Reply:
you need to install it as devDep so we can run the test

Reply:
Oh shoot 🤦. Sorry. Added!